### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by MdechampG**
Everything in the title

Following #252 and #253

## Summary by CodeRabbit

- **Documentation**
	- Corrected typos and improved clarity in the descriptions of governance and zone credentials within the schema documentation.
- **Bug Fixes**
	- Fixed a typo in the "Governance Text Credential" description.
	- Corrected the pluralization of "credential" in the zone description to ensure consistency and clarity.

### CODE REVIEW

File: docs/schemas/credential-zone-description.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Here are the namespaces used in this schema:
 
 Zone Description credentials deliver an extensive overview of the essential information about a zone. This typically comprises details like the zone's title, description, and tags.
 
-The primary objective of this credentials is to provide a broad overview of the zone, facilitating users' comprehension of its purpose and potential use cases.
+The primary objective of these credentials is to provide a broad overview of the zone, facilitating users' comprehension of its purpose and potential use cases.
 
 ### Examples
```
[the remark this answers is not in the document]
Comment (ccamel) [reply docs/schemas/credential-zone-description.md:35]: Nope. The opening parenthesis must be on the next line, as it is part of the content.

**Review by ccamel — APPROVED** [review]
LGTM! :+1:

File: docs/schemas/credential-governance-text.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Here are the namespaces used in this schema:
...
+The Governance Text Credential applies to all resources in the Dataverse for which rules must be followed. Such Governances are categorized into three main types as recognized by the [OKP4 protocol](https://okp4.network):
 
 - Zone Rules: Rules establish the boundaries of a specific area in the Dataverse, termed a Zone, within which governance is enforced. Each Zone is governed by its unique set of Rules tailored to its particular function or purpose.
 - Resource Consent: Consents are agreements associated with the use of Resources within Zones. By resources, we mean Digital Resources (e.g. datasets) or Digital Services. It goes beyond simple approval and encompasses the permissions and restrictions on resource owners' access, usage, sharing, management and handling of their resources. It allows parties to define boundaries and establish terms for others to interact with their digital resources. It's a crucial aspect of governance, ensuring resources are used appropriately per the relevant parties' expressed will and intentions.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/schemas/credential-governance-text.md:35]: We will add a glossary later, thanks.

**Comment by MdechampG** [comment]
@ccamel I need your help for the last lint problem please 🙏🏻

**Comment by ccamel** [comment]
@coderabbitai pause

**Comment by ccamel** [comment]
@MdechampG All is Charlie Mike.🫡

**Pull request merged**